pages/events/_id/edit.vue

```diff
@@ -30,7 +30,7 @@
         :fields="eventFields"
```

Comment:
What about form fields labels? Form is rendered based on `eventFields` array. See #114 as an example, it was similar task, that conains form.

Reply:
I had concerns about his task. The form-fields. 
Thank you for pointing me out the example. 
Working on it 🐱‍💻